src/DynamoCoreWpf/Views/HomePage/HomePage.xaml.cs

```diff
@@ -143,14 +145,17 @@ private async void UserControl_Loaded(object sender, System.Windows.RoutedEventA
             PathHelper.CreateFolderIfNotExist(userDataDir.ToString());
             var webBrowserUserDataFolder = userDataDir.Exists ? userDataDir : null;
 
+            var userDataFolder = CreateUniqueUserDataFolder(webBrowserUserDataFolder.FullName);
```

Comment:
this has not been a problem with other vebview2 tests...
Are you certain this solves your test issues ?
Also maybe we should just create and cleanup these unique folders only during testing ? either use the testing flag...or make these APIs accessible to the test project and call them directly from there.

Reply:
For the time being, testing things locally, I didn't run on any sporadic test failures as before. 

I was also thinking that maybe this is only necessary under test environment, so if the build completes correctly, I can make the necessary changes.